Wiki - Stories - Universal Rendering is missing a key ingredient for lazy loading

The [final part of the first step](https://github.com/angular/angular-cli/wiki/stories-universal-rendering#srcappappservermodulets) for the universal rendering is missing a very key part that is necessary for lazy loading to work correctly on the server.

Specifically app.server.module.ts should look like this:

```
import { NgModule } from '@angular/core';
import { ServerModule } from '@angular/platform-server';
import { ModuleMapLoaderModule } from '@nguniversal/module-map-ngfactory-loader';

import { AppModule } from './app.module';
import { AppComponent } from './app.component';

@NgModule({
  imports: [
    AppModule,
    ServerModule,
    ModuleMapLoaderModule
  ],
  bootstrap: [AppComponent],
})
export class AppServerModule { }
```


Note **ModuleMapLoaderModule** is necessary, otherwise node will throw an error every time a page that is supposed to be lazy loaded is attempted. An error akin to: ERROR: Cannot find module './lazy/lazy.module.ngFactory'. 

Interestingly this would be difficult to catch if you are not reading the node output since the pages still load only they are being rendered on the client, defeating the whole server side rendering purpose.

--- a/docs/documentation/stories/universal-rendering.md
+++ b/docs/documentation/stories/universal-rendering.md
@@ -55,9 +55,14 @@ This example places it alongside `app.module.ts` in a file named `app.server.mod
 
 ### src/app/app.server.module.ts:
 
+You can see here we're simply Importing everything from AppModule, followed by ServerModule.
+
+> One important thing to Note: We need `ModuleMapLoaderModule` to help make Lazy-loaded routes possible during Server-side renders with the Angular-CLI.
+
 ```typescript
 import {NgModule} from '@angular/core';
 import {ServerModule} from '@angular/platform-server';
+import {ModuleMapLoaderModule} from '@nguniversal/module-map-ngfactory-loader';
 
 import {AppModule} from './app.module';
 import {AppComponent} from './app.component';
@@ -67,7 +72,8 @@ import {AppComponent} from './app.component';
     // The AppServerModule should import your AppModule followed
     // by the ServerModule from @angular/platform-server.
     AppModule,
-    ServerModule,
+    ServerModule, 
+    ModuleMapLoaderModule // <-- *Important* to have lazy-loaded routes work
   ],
   // Since the bootstrapped component is not inherited from your
   // imported AppModule, it needs to be repeated here.

